src/lib/meta-head.ts

```diff
@@ -0,0 +1,64 @@
+/**
+ * Generate document head object for Nuxt.
+ *
+ * @see https://nuxtjs.org/api/pages-head
+ *
+ * @param {Object} meta
+ * @param {String} meta.title
+ * @param {String} [meta.description]
+ * @param {Object} [meta.image]
+ * @param {String} [meta.image.url]
+ * @param {Number} [meta.image.width]
+ * @param {Number} [meta.image.height]
+ */
+export default function ({
+  title,
+  description,
+  image,
+}: {
+  title: string;
+  description?: string;
+  image?: {
+    url: string;
+    width?: number;
+    height?: number;
+  };
+}) {
+  const meta = [{ hid: "og:title", property: "og:title", content: title }] as {
+    hid: string;
+    property?: string;
+    name?: string; //TODO: all good here?
```

Comment:
what do you mean?

Reply:
I sincerly do not remember. But since `useHead` / `meta` does not work properly in Nuxt 3 creating duplicate tags, I rewrote it to `useSeoMeta` and checked the result, which seems to be OK now. Thus the TODO is gone :)